Snowplow/Internal/Tracker/SPTrackerEvent.m

```diff
@@ -29,13 +29,14 @@ @implementation SPTrackerEvent
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations" // to ignore warnings for deprecated methods that we are forced to use until the next major version release
 
-- (instancetype)initWithEvent:(SPEvent *)event {
+- (instancetype)initWithEvent:(SPEvent *)event stateCopy:(NSDictionary<NSString *,SPStateFuture *> *)stateCopy {
```

Comment:
I appreciate you want a copy to be passed in here, so I get calling it `stateCopy`, seems a bit unfortunately to need to expose that detail here though - not sure on how to improve it besides taking a copy inside the function instead.